src/coreclr/System.Private.CoreLib/src/Internal/Runtime/InteropServices/ComponentActivator.cs

```diff
@@ -139,6 +151,7 @@ private static IsolatedComponentLoadContext GetIsolatedComponentLoadContext(stri
             return alc;
         }
 
+        [RequiresUnreferencedCode("The trimmer might remove methods")]
```

Comment:
This is a public facing message (the trimmer will report it if the feature switch is enabled). So it should be descriptive and actionable (well - in this case the actionable may not be exact).

Reply:
It might be better to add `RequiresUnreferencedCode` onto the entry points (`GetFunctionPointer` in this case). It would be cleaner. Otherwise the name of the internal method will show up in the message - which might be a bit weird. But we will have to verify that linker actually reports the warning in that case.